index.bs

```diff
@@ -1266,7 +1266,9 @@ To <dfn>process an attributionsrc response</dfn> given a [=suitable origin=]
                 [=create an attribution trigger=] with |triggerHeader|
                 |destinationSite|, |reportingOrigin|, |privateStateTokens|, and
                 |context|'s [=current wall time=].
-            1. If |trigger| is not null, [=trigger attribution=] with |trigger|.
+            1. If |trigger| is not null:
+                1. [=maybe defer trigger attribution=] with |trigger|.
```

Comment:
```suggestion
                1. [=Maybe defer trigger attribution=] with |trigger|.
```

Reply:
Other algorithms defined are lowercased, I'd be inclined to keep it lowercased.